packages/block-editor/src/components/block-list/block.js

```diff
@@ -534,14 +472,9 @@ function BlockListBlock( {
 		<IgnoreNestedEvents
 			id={ blockElementId }
 			ref={ wrapper }
-			onMouseOver={ maybeHover }
-			onMouseOverHandled={ hideHoverEffects }
```

Comment:
I think this was the only usage of the "Handled" behavior for `IgnoreNestedEvents`, so it could be removed. Not sure that it will make a ton of difference, but at least simplifying a bit.

Effectively would amount to a revert of 0d0c8e0d55849529227df3540ad35ff3edf5be5b (from #5658)